pkg/standalone/common.go

```diff
@@ -6,15 +6,31 @@ import (
 	"runtime"
 )
 
-// getDefaultComponentsFolder returns the hidden .components folder created at init time
-func GetDefaultComponentsFolder() string {
-	const daprDirName = ".dapr"
-	const componentsDirName = "components"
-	daprDirPath := os.Getenv("HOME")
+const (
+	defaultDaprDirName       = ".dapr"
+	DefaultComponentsDirName = "components"
+	defaultConfigDirName     = "config"
+)
+
+func getHomeFolder() string {
+	homePath := os.Getenv("HOME")
 	if runtime.GOOS == daprWindowsOS {
-		daprDirPath = os.Getenv("USERPROFILE")
+		homePath = os.Getenv("USERPROFILE")
+	}
+	return homePath
+}
+
+// GetDefaultFolderPath returns the default requested path to the requested path
+func GetDefaultFolderPath(dirName string) string {
+	homePath := getHomeFolder()
+	if dirName == defaultDaprDirName {
+		return path_filepath.Join(homePath, defaultDaprDirName)
 	}
+	return path_filepath.Join(homePath, defaultDaprDirName, dirName)
+}
 
-	defaultComponentsPath := path_filepath.Join(daprDirPath, daprDirName, componentsDirName)
-	return defaultComponentsPath
+func GetDefaultConfigFilePath() string {
```

Comment:
To make this consistent, either remove this func or add a GetDefaultComponentsPath()?